packages/ra-core/src/controller/useListParams.ts

```diff
@@ -56,6 +56,7 @@ const defaultSort = {
 };
 
 const defaultParams = {};
+let tempParams;
```

Comment:
This should be a ref

Reply:
Why not making it a ref then?

Reply:
I added the note for myself. it's a ref now.